ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -18,10 +19,14 @@ spec:
       workingDir: $(workspaces.results.path)
       script: |
         #! /usr/bin/env bash
+
         # Did the tests pass?
-        # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
         # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_RESULTS=$(params.preflight_results)
+        CHECK_RESULTS=$(cat PREFLIGHT_RESULTS | jq '.passed' | grep .)
+        if [ $CHECK_RESULTS == "false"]; then
```

Comment:
this "if" doesn't work

Reply:
Oh Yeah, I am comparing the wrong value. Thanks for dragging my attention here.

Reply:
NVMD, I got your statement.

Reply:
it wasn't working because of syntax error- bracket without space before it. Thanks for resolving it, please test your changes before submitting them to code review.